app/dao/templates_dao.py

```diff
@@ -89,70 +95,98 @@ def dao_redact_template(template, user_id):
 
 
 def dao_get_template_by_id_and_service_id(template_id, service_id, version=None) -> Template:
-    if version is not None:
-        return TemplateHistory.query.filter_by(
-            id=template_id,
-            hidden=False,
-            service_id=service_id,
-            version=version).one()
-    return Template.query.filter_by(id=template_id, hidden=False, service_id=service_id).one()
+    if version is None:
+        stmt = select(Template).where(
+            Template.id == template_id,
+            Template.hidden.is_(False),
+            Template.service_id == service_id
+        )
+    else:
+        stmt = select(TemplateHistory).where(
+            TemplateHistory.id == template_id,
+            TemplateHistory.hidden.is_(False),
```

Comment:
Better way?

Reply:
Note sure what "better way" means? What did we agree on? Needs changes?

Reply:
No change.  is_ is the correct syntax to test booleans.